app/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,80 @@
+// import { useState } from 'react';
+import { NavLink } from '@remix-run/react';
+import { Button } from '~/components/ui/button';
+
+
+
+interface NavbarProps {
+  isUserAuthenticated: boolean;
+}
+
+const Navbar = ({ isUserAuthenticated }: NavbarProps) =>  {
+
+  // const [mobileMenuOpen, setMobileMenuOpen] = useState(false);  // For Mobile Nav SideBar
+
+  const navigation = [
+    { name: 'Home', to: '/' },
+    { name: 'Pricing', to: '/pricing' },
```

Comment:
Create a data folder where you keep this navigation data and import it to this file